src/rules/__tests__/no-expect-resolves.test.ts

```diff
@@ -0,0 +1,24 @@
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import rule from '../no-expect-resolves';
+
+const ruleTester = new TSESLint.RuleTester({
+  parserOptions: {
+    ecmaVersion: 2017,
+  },
+});
+
+ruleTester.run('no-expect-resolves', rule, {
```

Comment:
this rule should probably have more tests, but I ran it against Jest and found no errors (at least no false positives and no crashes)